### YamlMime:ManagedReference
items:
  - uid: com.microsoft.azure.eventhubs._event_hub_exception
    id: _event_hub_exception
    parent: com.microsoft.azure.eventhubs
    children:
      - 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final String)'
      - 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final String,final Throwable)'
      - 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final Throwable)'
      - com.microsoft.azure.eventhubs._event_hub_exception.getContext()
      - com.microsoft.azure.eventhubs._event_hub_exception.getIsTransient()
      - com.microsoft.azure.eventhubs._event_hub_exception.getMessage()
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: EventHubException
    nameWithType: EventHubException
    fullName: com.microsoft.azure.eventhubs.EventHubException
    type: Class
    source:
      remote: &ref_0
        path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
        branch: master
        repo: 'https://github.com/Azure/azure-event-hubs-java'
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 11
    package: com.microsoft.azure.eventhubs
    summary: <p>これは、基本例外をサービス バスは、すべてのエラー ケースを生成します。 </p>
    syntax:
      content: public class EventHubException
    inheritance:
      - java.lang.Object
      - Exception
  - uid: 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final String)'
    id: 'EventHubException(final boolean,final String)'
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: 'EventHubException(final boolean isTransient, final String message)'
    nameWithType: 'EventHubException.EventHubException(final boolean isTransient, final String message)'
    fullName: 'com.microsoft.azure.eventhubs.EventHubException.EventHubException(final boolean isTransient, final String message)'
    overload: com.microsoft.azure.eventhubs._event_hub_exception.EventHubException*
    type: Constructor
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 24
    package: com.microsoft.azure.eventhubs
    syntax:
      content: 'public EventHubException(final boolean isTransient, final String message)'
      parameters:
        - id: isTransient
          type: 4dda0764
        - id: message
          type: 3d93591e
  - uid: 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final String,final Throwable)'
    id: 'EventHubException(final boolean,final String,final Throwable)'
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: 'EventHubException(final boolean isTransient, final String message, final Throwable cause)'
    nameWithType: 'EventHubException.EventHubException(final boolean isTransient, final String message, final Throwable cause)'
    fullName: 'com.microsoft.azure.eventhubs.EventHubException.EventHubException(final boolean isTransient, final String message, final Throwable cause)'
    overload: com.microsoft.azure.eventhubs._event_hub_exception.EventHubException*
    type: Constructor
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 36
    package: com.microsoft.azure.eventhubs
    syntax:
      content: 'public EventHubException(final boolean isTransient, final String message, final Throwable cause)'
      parameters:
        - id: isTransient
          type: 4dda0764
        - id: message
          type: 3d93591e
        - id: cause
          type: fa00b19f
  - uid: 'com.microsoft.azure.eventhubs._event_hub_exception.EventHubException(final boolean,final Throwable)'
    id: 'EventHubException(final boolean,final Throwable)'
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: 'EventHubException(final boolean isTransient, final Throwable cause)'
    nameWithType: 'EventHubException.EventHubException(final boolean isTransient, final Throwable cause)'
    fullName: 'com.microsoft.azure.eventhubs.EventHubException.EventHubException(final boolean isTransient, final Throwable cause)'
    overload: com.microsoft.azure.eventhubs._event_hub_exception.EventHubException*
    type: Constructor
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 30
    package: com.microsoft.azure.eventhubs
    syntax:
      content: 'public EventHubException(final boolean isTransient, final Throwable cause)'
      parameters:
        - id: isTransient
          type: 4dda0764
        - id: cause
          type: fa00b19f
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getContext()
    id: getContext()
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: getContext()
    nameWithType: EventHubException.getContext()
    fullName: ErrorContext com.microsoft.azure.eventhubs.EventHubException.getContext()
    overload: com.microsoft.azure.eventhubs._event_hub_exception.getContext*
    type: Method
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 62
    package: com.microsoft.azure.eventhubs
    syntax:
      content: public ErrorContext getContext()
      return:
        type: com.microsoft.azure.eventhubs._error_context
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getIsTransient()
    id: getIsTransient()
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: getIsTransient()
    nameWithType: EventHubException.getIsTransient()
    fullName: boolean com.microsoft.azure.eventhubs.EventHubException.getIsTransient()
    overload: com.microsoft.azure.eventhubs._event_hub_exception.getIsTransient*
    type: Method
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 57
    package: com.microsoft.azure.eventhubs
    summary: <p>例外が、一時的なエラーかを示すブール値。 </p>
    syntax:
      content: public boolean getIsTransient()
      return:
        type: 4fc6e284
        description: <p>ユーザーがその他の介入なし例外を生成した操作を再試行できる場合に true を返します。 </p>
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getMessage()
    id: getMessage()
    parent: com.microsoft.azure.eventhubs._event_hub_exception
    href: com.microsoft.azure.eventhubs._event_hub_exception.yml
    langs:
      - java
    name: getMessage()
    nameWithType: EventHubException.getMessage()
    fullName: String com.microsoft.azure.eventhubs.EventHubException.getMessage()
    overload: com.microsoft.azure.eventhubs._event_hub_exception.getMessage*
    type: Method
    source:
      remote: *ref_0
      path: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/EventHubException.java
      startLine: 43
    package: com.microsoft.azure.eventhubs
    syntax:
      content: public String getMessage()
      return:
        type: '26831127'
references:
  - uid: 4dda0764
    spec.java:
      - name: final boolean
        fullName: final boolean
  - uid: 3d93591e
    spec.java:
      - name: final String
        fullName: final String
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.EventHubException*
    name: EventHubException
    nameWithType: EventHubException.EventHubException
    fullName: com.microsoft.azure.eventhubs.EventHubException.EventHubException
    package: com.microsoft.azure.eventhubs
  - uid: fa00b19f
    spec.java:
      - name: final Throwable
        fullName: final Throwable
  - uid: '26831127'
    spec.java:
      - name: String
        fullName: String
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getMessage*
    name: getMessage
    nameWithType: EventHubException.getMessage
    fullName: String com.microsoft.azure.eventhubs.EventHubException.getMessage
    package: com.microsoft.azure.eventhubs
  - uid: 4fc6e284
    spec.java:
      - name: boolean
        fullName: boolean
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getIsTransient*
    name: getIsTransient
    nameWithType: EventHubException.getIsTransient
    fullName: boolean com.microsoft.azure.eventhubs.EventHubException.getIsTransient
    package: com.microsoft.azure.eventhubs
  - uid: com.microsoft.azure.eventhubs._event_hub_exception.getContext*
    name: getContext
    nameWithType: EventHubException.getContext
    fullName: ErrorContext com.microsoft.azure.eventhubs.EventHubException.getContext
    package: com.microsoft.azure.eventhubs
  - uid: com.microsoft.azure.eventhubs._authorization_failed_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._authorization_failed_exception.yml
    name: AuthorizationFailedException
    nameWithType: AuthorizationFailedException
    fullName: com.microsoft.azure.eventhubs.AuthorizationFailedException
    type: Class
    summary: |-
      <p>承認が目的の操作を実行するユーザーのアクセス許可の承認中にエラーが発生した場合に障害が発生した例外がスローされます。 この例外が発生したときにユーザーが接続文字列でトークンまたはキーが提供されるかどうかを確認する必要があります (例: いずれかに渡す<xref uid="com.microsoft.azure.eventhubs._event_hub_client.createFromConnectionString(final String)" data-throw-if-not-resolved="false">com.microsoft.azure.eventhubs.EventHubClient#createFromConnectionString(String)</xref>) 有効ですが、あり (受信呼び出しなどが必要なリッスン キーまたはトークンに関連付けられた要求)、目的の操作の権利を正しく実行されます。</p>
      <p></p>
    syntax:
      content: public class AuthorizationFailedException extends EventHubException
  - uid: com.microsoft.azure.eventhubs._communication_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._communication_exception.yml
    name: CommunicationException
    nameWithType: CommunicationException
    fullName: com.microsoft.azure.eventhubs.CommunicationException
    type: Class
    summary: |-
      <p>クライアント側の接続に問題がある場合は、この例外がスローされます。 ときにこの例外のユーザーの受信は、サービスにクライアント接続設定を確認する必要があります。<itemizedlist><listitem><p>正しいホスト名およびエンドポイントで使用されるポート番号を確認します。 </p></listitem><listitem><p>Amqp ポートをブロックするすべての可能なプロキシ設定を確認します。 </p></listitem><listitem><p>Amqp ポートをブロックするファイアウォールの設定の確認します。 </p></listitem><listitem><p>一般的なネットワーク接続の問題と、ネットワークの待機時間を確認します。 </p></listitem></itemizedlist></p>
      <p></p>
    syntax:
      content: public class CommunicationException extends EventHubException
  - uid: com.microsoft.azure.eventhubs._illegal_entity_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._illegal_entity_exception.yml
    name: IllegalEntityException
    nameWithType: IllegalEntityException
    fullName: com.microsoft.azure.eventhubs.IllegalEntityException
    type: Class
    summary: |-
      <p>この例外は、次の理由。<itemizedlist><listitem><p>エンティティ ユーザーが接続しようとしたときに存在しません </p></listitem><listitem><p>接続するエンティティのユーザーが本人が無効になっています </p></listitem></itemizedlist></p>
      <p></p>
    syntax:
      content: public class IllegalEntityException
  - uid: com.microsoft.azure.eventhubs._operation_cancelled_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._operation_cancelled_exception.yml
    name: OperationCancelledException
    nameWithType: OperationCancelledException
    fullName: com.microsoft.azure.eventhubs.OperationCancelledException
    type: Class
    summary: <p>この例外は、基になる Amqp 層が異常なリンクを検出すると、中止または予期しない方法で接続の切断します。 </p>
    syntax:
      content: public class OperationCancelledException
  - uid: com.microsoft.azure.eventhubs._payload_size_exceeded_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._payload_size_exceeded_exception.yml
    name: PayloadSizeExceededException
    nameWithType: PayloadSizeExceededException
    fullName: com.microsoft.azure.eventhubs.PayloadSizeExceededException
    type: Class
    summary: |-
      <p>ユーザーが、イベント データやサービスによって定義されている許可されたペイロードのサイズを超えました仲介型メッセージを送信しようとした場合、この例外がスローされます。 注バッチのシナリオに制限を送信するには、可能なバッチのオーバーヘッドを含めることができます。</p>
      <p></p>
    syntax:
      content: public class PayloadSizeExceededException
  - uid: com.microsoft.azure.eventhubs._quota_exceeded_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._quota_exceeded_exception.yml
    name: QuotaExceededException
    nameWithType: QuotaExceededException
    fullName: com.microsoft.azure.eventhubs.QuotaExceededException
    type: Class
    syntax:
      content: public class QuotaExceededException
  - uid: com.microsoft.azure.eventhubs._receiver_disconnected_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._receiver_disconnected_exception.yml
    name: ReceiverDisconnectedException
    nameWithType: ReceiverDisconnectedException
    fullName: com.microsoft.azure.eventhubs.ReceiverDisconnectedException
    type: Class
    summary: |-
      <p>次の理由のいずれかの理由、EventHubReceiver は接続が切断されるときに、この例外がスローされます。<itemizedlist><listitem><p>エポック レシーバーがある場合に、イベント ハブ パーティションに非エポック レシーバーを接続するユーザーの試行は、パーティションに接続します。 </p></listitem><listitem><p>エポック受信者を特定のパーティションを使用しているより高いエポック値を持つ別のエポック受信アダプターが同じパーティションに接続します。 </p></listitem></itemizedlist>ユーザーは、か、すべてのコード非エポック受信機を使用している 1 つだけのエポック受信機が時間内の任意の時点で、特定のパーティションの処理があることを確認することを確認してください。</p>
      <p></p>
    syntax:
      content: public class ReceiverDisconnectedException
  - uid: com.microsoft.azure.eventhubs._server_busy_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._server_busy_exception.yml
    name: ServerBusyException
    nameWithType: ServerBusyException
    fullName: com.microsoft.azure.eventhubs.ServerBusyException
    type: Class
    summary: |-
      <p>現在のエンティティのアクティビティがサービスに過剰な負荷を挿入がサーバーのビジー状態である例外がスローされます。 この例外が発生したときにユーザーとエンティティの再試行、および実行時の操作は、もう一度の前に、少なくとも 4 秒を待機する必要があります。</p>
      <p></p>
    syntax:
      content: public class ServerBusyException
  - uid: com.microsoft.azure.eventhubs._timeout_exception
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._timeout_exception.yml
    name: TimeoutException
    nameWithType: TimeoutException
    fullName: com.microsoft.azure.eventhubs.TimeoutException
    type: Class
    summary: |-
      <p>操作があらかじめ設定された時間制限を超えたときに、この例外がスローされます。 ユーザーは、接続はクライアント プロセスとサービス間で正常を確認する必要があります。</p>
      <p></p>
    syntax:
      content: public class TimeoutException
  - uid: com.microsoft.azure.eventhubs._error_context
    parent: com.microsoft.azure.eventhubs
    href: com.microsoft.azure.eventhubs._error_context.yml
    name: ErrorContext
    nameWithType: ErrorContext
    fullName: com.microsoft.azure.eventhubs.ErrorContext
    type: Class
    syntax:
      content: public class ErrorContext